includes/views/object-content-actions.php

```diff
@@ -0,0 +1,96 @@
+<?php
+	/**
+	 * The template to render rubric actions.
+	 * The template requires $object_id and $object_type to be passed in.
+	 *
+	 * @package ubc_wp_vote
+	 */
+
+if ( ! isset( $object_id ) || ! isset( $object_type ) ) {
+	return;
+}
+
+require_once 'wp-admin/includes/template.php';
+
+$object_id = intval( $object_id );
+$object_type = sanitize_text_field( $object_type );
+
+$current_user_thumbs_up = \UBC\CTLT\WPVote\WP_Vote::get_object_current_user_up_vote(
+	array(
+		'object_type' => $object_type,
+		'object_id' => $object_id,
+	)
+);
+
+$total_thumbs_up = \UBC\CTLT\WPVote\WP_Vote::get_object_current_user_up_vote(
+	array(
+		'object_type' => $object_type,
+		'object_id' => $object_id,
+	)
+);
+
+$current_user_thumbs_down = \UBC\CTLT\WPVote\WP_Vote::get_object_current_user_down_vote(
+	array(
+		'object_type' => $object_type,
+		'object_id' => $object_id,
+	)
+);
+
+$current_user_rating = \UBC\CTLT\WPVote\WP_Vote::get_object_current_user_rate(
+	array(
+		'object_type' => $object_type,
+		'object_id' => $object_id,
+	)
+);
+
+$total_rating = \UBC\CTLT\WPVote\WP_Vote::get_object_rate_average(
+	array(
+		'object_type' => $object_type,
+		'object_id' => $object_id,
+	)
+);
+
+$total_thumbs_up = \UBC\CTLT\WPVote\WP_Vote::get_object_total_up_vote(
+	array(
+		'object_type' => $object_type,
+		'object_id' => $object_id,
+	)
+);
+
+?>
+<?php if ( is_user_logged_in() ) : ?>
```

Comment:
Fine for now, but let's explore a way to break this HTML out into their own template files. Makes reading/updating/editing/overwriting much easier.